dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/spi/package-info.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+/**
+ *
+ * <h1> OpenLiberty Microprofile Telemetry Interface Design.</h1>
```

Comment:
```suggestion
 * <h1> Open Liberty MicroProfile Telemetry Interface</h1>
```
Open Liberty is two words, not one.  Should the word `Design` removed and the period?